Hello-World Project Module One

AleWaweru opened the pull request:
created a readme file
I configured Linters
I created HTML and CSS

AleWaweru committed:
Readme Updates

AleWaweru committed:
Merge branch 'add-newBranch' of github.com:AleWaweru/Hello-World into add-newBranch

AleWaweru committed:
edited ReadMe

aamirkhan2478 reviewed (changes requested):
Hi @AleWaweru , 

![](https://media3.giphy.com/media/RcZlXVml8oKtoGQeqe/giphy.gif?cid=ecf05e47cw5pjd1gapdzk1t3vp0n73caujoqxw3vkcu8ssng&rid=giphy.gif&ct=g)

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights
✔️ Good Effort
✔️ Your design is good.

 ## Required Changes ♻️
- [ ] Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise. :pray: 
- [ ] Please create a test.md file with the text "This file should be ignored by git". :slightly_smiling_face: 

![Screenshot from 2023-03-22 23-55-01](https://user-images.githubusercontent.com/45226848/227010435-5bd1adac-ca3e-4dfd-a659-802311ddcb7b.png)
- [ ] Please add the `live demo` section in your `README` file its missing only `FAQ` is optional other sections are required sections so kindly add the `live demo` section. :+1: 

![Screenshot from 2023-03-22 23-54-23](https://user-images.githubusercontent.com/45226848/227011597-3c7c605b-0795-43f1-aa8d-d88363ebf348.png)
- [ ] Please remove `Server` and `database` inside the Tech stack section because in this project you are not using any server and database so kindly remove them and add `HTML` and `CSS` inside `Client`  because you are using `HTML` and `CSS` in this project so kindly fix this also. :slightly_smiling_face: 

![Screenshot from 2023-03-22 23-55-36](https://user-images.githubusercontent.com/45226848/227011772-53070d17-3a42-4805-abce-600724cd3848.png)
- [ ] Your `MIT` is not working kindly fix this by going to the microverse provided `README template repo` and copying the `MIT.md` file and past it into your project and changing `[APP TYPE]` to the `website` and also change the path of your link like that `[MIT](./MIT.md)` and then check it's working or not if it's not working check your path again and fix it. :+1: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AleWaweru committed:
Update README.md

Made the required Changes on my ReadMe file.

AleWaweru committed:
Changes on ReadMe File and added MIT.md file

AleWaweru committed:
Merge branch 'add-newBranch' of github.com:AleWaweru/Hello-World into add-newBranch

AleWaweru commented:
Made the Changes required
1.test.md file already exists inside the .gitignore file which technically was to be ignored when on github
2. Added the live Demo Section as instructed in the ReadMe file
3. Removed the server and database section and included HTML and CSS into the Client Section inthe ReadMe file
4. Added the MIT.md file into my project which is working perfectly as required

#I am using the 2nd pull request because I encountered challenges using the first pull request.

wayungi reviewed (approved):
Hi @AleWaweru , 

![image](https://user-images.githubusercontent.com/99656687/220082459-c1b049c7-5498-42e7-942b-ec7a05d3e603.png)

Great work, your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**Remember: Every finish line 🏁  is the beginning of a new race 🐎** 

## Highlights
- [ ] Good README ✔️ 
- [ ] All linter checks passing ✔️ 
- [ ]  Correct GitHub flow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
